ui/raidboss/data/06-ew/dungeon/ktisis_hyperboreia.txt

```diff
@@ -77,23 +82,26 @@ hideall "--sync--"
 1077.4 "Pyric Sphere" sync / 1[56]:[^:]*:Pyric Sphere:6490:/
 1079.8 "--sync--" sync / 1[56]:[^:]*:Ladon Lord:6485:/
 1082.5 "Pyric Sphere" sync / 1[56]:[^:]*:Pyric Sphere:6491:/
-1088.9 "Pyric Breath #1" #sync / 1[56]:[^:]*:Ladon Lord:(6486|6487|648A|648B):/
-1091.0 "Pyric Breath #2" #sync / 1[56]:[^:]*:Ladon Lord:(6486|6487|648A|648B):/
+1081.9 "--rotate--"
```

Comment:
The tiniest of nits, but perhaps swap the Rotate and Pyric Sphere lines to maintain timestamp order, if the Rotate cast isn't intended to *be* the Rotate?

Reply:
Ah, thanks! Yeah, the boss rotates as the sphere lines go off.